assets/css/main.css

```diff
@@ -0,0 +1,507 @@
+.bolder {
+  font-weight: 700;
+}
+
+.sticky {
+  position: sticky;
+  top: 50px;
+}
+
+.fa {
+  font-size: 8px;
+  vertical-align: middle;
+}
+
+header {
+  z-index: 1;
+}
+
+img:hover {
+  cursor: pointer;
+}
+
+.header nav {
+  width: 100%;
+  height: 50px;
+  padding: 15px 15px;
+  background-color: #f72210;
+  justify-content: space-between;
+}
+
+.header nav .brand {
+  width: 132px;
+}
+
+footer .social-links svg {
+  width: 236px;
+  fill: #f72210;
+}
+
+.header nav .brand svg {
+  fill: white;
+}
+
+button.subscribe-btn {
+  border: 0;
+  width: 93px;
+  font-size: 0.85em;
+  border-radius: 16px;
+  padding: 1px 10px;
+  font-weight: 700;
+  background-color: #ffe400;
+  display: inline-block;
+}
+
+.sub-content button {
+  background-color: #f72210;
+  color: white;
+  font-weight: 700;
+  width: 122px;
+}
+
+.submission form button {
+  background-color: #f61602;
+}
+
+.sponsored-section-2 .form button {
+  background-color: #f61602;
+}
```

Comment:
Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).

Reply:
Hi Oscar, thank you very much for reviewing my project. I checked the original page navbar and I saw that the user icon stays until 768px breakpoint, like this: 

Original:

![original](https://user-images.githubusercontent.com/61803988/84815330-7b9e7380-afd8-11ea-9b5e-30a4815b74b0.PNG)

Clone: 

![clone](https://user-images.githubusercontent.com/61803988/84815366-89ec8f80-afd8-11ea-8309-0f9f13863d8c.PNG)

I don't really know what to do. Should I make the change? or maybe its a bug of mine?
Please Oscar let me know what to do.

you are right I repeat my self too many times, I dried my code as much as I could, I'll pay more attention to that in my future projects =D